### PR TITLE
ADM-364 T9 - [frontend] [Metrics] Classification Settings

### PR DESCRIPTION
**Pull request by ZhiqiaoYang**
1.fix e2e test 
2.classification component display

### CODE REVIEW

File: frontend/src/components/Metrics/MetricsStep/index.tsx [hunk 1]
```diff
@@ -3,14 +3,20 @@ import { Crews } from '@src/components/Metrics/MetricsStep/Crews'
...
 export const MetricsStep = () => {
+  const requiredData = useAppSelector(selectMetrics)
   const users = useAppSelector(selectUsers)
   const targetFields = useAppSelector(selectTargetFields)
+
   return (
     <>
       <Crews options={users} title={'Crews Setting'} label={'Included Crews'} />
-      <Classification options={targetFields} title={'Classification Setting'} label={'Distinguished By'} />
+      {requiredData.length > 0 && requiredData.includes(REQUIRED_DATA_LIST[2]) && (
```
Comment (mengna-tw) [inline frontend/src/components/Metrics/MetricsStep/index.tsx:17]: maybe remove >0

File: frontend/__tests__/src/components/Metrics/MetricsStep/MetricsStep.test.tsx [hunk 2]
```diff
@@ -0,0 +1,31 @@
+import { render } from '@testing-library/react'
+import { MetricsStep } from '@src/components/Metrics/MetricsStep'
+import { Provider } from 'react-redux'
+import { setupStore } from '../../../utils/setupStoreUtil'
+
+import { updateMetrics } from '@src/context/config/configSlice'
+
+const store = setupStore()
+const setup = () => {
+  return render(
```
Comment (mengna-tw) [inline frontend/__tests__/src/components/Metrics/MetricsStep/MetricsStep.test.tsx:10]: maybe remove return

File: frontend/__tests__/src/components/Metrics/MetricsStep/MetricsStep.test.tsx [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+describe('MetricsStep', () => {
+  it('should show default crews setting', () => {
+    const { getByText, queryByText } = setup()
+
+    expect(getByText('Crews Setting')).toBeInTheDocument()
+    expect(queryByText('Classification Setting')).not.toBeInTheDocument()
+  })
+
+  it('should show Classification Setting when select classification in config page', async () => {
+    await store.dispatch(updateMetrics(['Classification']))
```
Comment (mengna-tw) [inline frontend/__tests__/src/components/Metrics/MetricsStep/MetricsStep.test.tsx:26]: maybe format this line

**Review by mengna-tw — COMMENTED** [review]
left some comments , feel free to fix them

**Review by mengna-tw — APPROVED** [review]

File: frontend/src/components/Metrics/MetricsStep/index.tsx (hunk 1 above)
Comment (tingwang1234) [reply frontend/src/components/Metrics/MetricsStep/index.tsx]: done

File: frontend/__tests__/src/components/Metrics/MetricsStep/MetricsStep.test.tsx (hunk 2 above)
Comment (tingwang1234) [reply frontend/__tests__/src/components/Metrics/MetricsStep/MetricsStep.test.tsx]: done

File: frontend/__tests__/src/components/Metrics/MetricsStep/MetricsStep.test.tsx (hunk 3 above)
Comment (tingwang1234) [reply frontend/__tests__/src/components/Metrics/MetricsStep/MetricsStep.test.tsx:30]: we want to dispatch 'Classification' to store first, and then to run test

**Review by mengna-tw — APPROVED** [review]

**Pull request merged**